pkg/operatormanager/operatormanager.go

```diff
@@ -683,3 +703,19 @@ func (m *OperatorManager) UpdateAllOperators(ctx context.Context) error {
 	m.log.Info("Done updating postgres operators in managed namespaces")
 	return nil
 }
+
+func perNamespaceSidecarsConfigMapName(cm *v1.ConfigMap) string {
```

Comment:
we should also talk about this, as already mentioned above. that looks too complicated to me?

Reply:
As above, use hard-coded `const` instead.